src/Cli/dotnet/Parser.cs

```diff
@@ -271,7 +271,7 @@ public override void Write(HelpContext context)
                     option.EnsureHelpName();
                 }
 
-                if (command.Name.Equals(NuGetCommandParser.GetCommand().Name))
+                if (command.Equals(NuGetCommandParser.GetCommand()) || command.Parents.Any(parent => parent == NuGetCommandParser.GetCommand()))
```

Comment:
this is not strictly related to this PR, but the nuget help won't show (SDK help hijacks it) unless I add this.